pandas/core/indexes/multi.py

```diff
@@ -1317,7 +1319,7 @@ def from_tuples(cls, tuples, sortorder=None, names=None):
             arrays = [[]] * len(names)
         elif isinstance(tuples, (np.ndarray, Index)):
             if isinstance(tuples, Index):
-                tuples = tuples._values
+                tuples = tuples._ndarray_values
```

Comment:
Why `ndarray_values` in this case? Because I think this code path can only work if it is an object index that contains tuples, so should not be different in this case.

Although that maybe raises the question: in such cases, where we know both `_values` and `_ndarray_values` will return the same ndarray (eg in all code only for numeric indexes), which one of the two do we take? (might be good to be consistent on this)